CoreAnalyticView/nbproject/project.xml

```diff
@@ -94,6 +94,14 @@
                         <specification-version>1.0</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
```

Comment:
So we can't do this @formalhaut69 because it means that the Analytic View becomes tied to the Attribute Editor. This becomes a problem should we decide to replace the Attribute Editor with a different implementation module; we can't because it will want to remove the Analytic View too.

![image](https://user-images.githubusercontent.com/39325530/89564235-0d4f9200-d860-11ea-97b8-06b9bcb89806.png)

FYI all of the views now are not dependant on one another so we gotta keep it that way.

Perhaps look at the implementation in `RemoveUnusedAttributesPlugin` for inspiration and you might need to create a new utility method that could live somewhere in the `Core Graph Framework` module so it can be used by views and any plugin in the future.

Reply:
Okay, no worries @arcturus2 . I'll try and figure out an alternate way to do this without making any new dependencies.